Data-Profiles.md

```diff
@@ -41,5 +41,57 @@ Yearly 26kb / data source
 
 ![Data Source Profiles RRA Edit](images/data-source-profiles-edit2.png)
 
+### Changing data profile averages 
+
+The Cacti gui allows you to change the averages on a specific Data source profile
+some setups require a higher resolution of data and you may want to change the averaging to
+keep the most data this will come with a cost of storage due to the bigger sized RRD file
+
+To do this follow these steps
+
+#### Method 1: New system without any hosts
+
+1.) Login to cacti
+
+2.) Click on Presets >> Data Profiles
+
+3.) Select the profile you want to edit i.e 1 minute collection
+
+4.) Select the period which you want to edit i.e weekly 
+
+5.) if you wish to change the average select the aggregate level for 1 minute you will be able to go as low as 5 Minutes
+
+6.) you will need to edit the Rows amount to match the data retention you want to set after this period the data will be 
+averaged out to the monthly graph
+
+7.) save config and generate graphs
+
+
+#### Method 2: Existing device on system ( Remove exisitng graphs)
+
+This is a more data destructive route and it will depend on your situation
+If you have graphs already on cacti you will not be able to use method 1
+One option if you do not care about the graphs is to either remove the graphs via the gui
+or run the following script /cli/php remove_graphs.php with the --all function which will REMOVE ALL THE GRAPHS
+You can then follow the steps outlined in Method 1 to change the profile how you like
+
+Method 3: Existing device on system ( Keep Graphs)
```

Comment:
Seems to be missing the appropriate titling prefix

Reply:
that should have been removed actually I am working on method 3 thanks!